keras_nlp/tokenizers/word_piece_tokenizer.py

```diff
@@ -307,22 +313,41 @@ def __init__(
                 )
 
         super().__init__(**kwargs)
+        
+        if vocabulary_size is not None:
+            if isinstance(vocabulary_size, int) == False:
+                raise ValueError(
+                    "vocabulary size must be int. "
+                    f"Recieved: type {type(vocabulary_size)}"
+                )
```

Comment:
Can be made simpler.

```
if vocabulary_size is not None and not isinstance(vocabulary_size, int):
    raise ValueError("...")
```

On second thoughts, we don't do type checks anywhere in KerasNLP. So, we can probably remove this entirely?